src/sentry/models/project.py

```diff
@@ -249,12 +249,19 @@ def get_member_alert_settings(self, user_option):
         :param user_option: alert option key, typically 'mail:alert'
         :return: A dictionary in format {<user_id>: <int_alert_value>}
         """
-        from sentry.models import UserOption
+        from sentry.models import UserOption, User
+
+        member_alert_disabled = {}
+        for user in User.objects.get_from_projects(self.organization.id, [self]):
```

Comment:
It will be more efficient to get all the users as a list, and fetch the `UserOption` rows at once, then iterate through those. That way we avoid making a query per user